### PR TITLE
Component migration   tooltip

### PR DESCRIPTION
**Pull request by lordbyronknox**

### CODE REVIEW

File: src/tooltip/index.tsx [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+ * ----------------------------------------------- */
+
+export type TriggerButtonProps = RACButtonProps;
+
+/**
+ * A field button is a button that is intended to be used inside a `Group` component
+ * to add additional functionality to a field. The `slot` prop is used to connect the
+ * button to the field.
+ */
+export const TooltipFieldButton = forwardRef<
```
Comment (alex-mcgovern) [inline src/tooltip/index.tsx:34]: I feel the naming is off — maybe `TooltipTriggerButton` instead?

File: src/tooltip/index.tsx [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+    HTMLButtonElement,
+    TriggerButtonProps
+>((props: RACButtonProps, ref) => {
+    return (
+        <RACButton
```
Comment (alex-mcgovern) [inline src/tooltip/index.tsx:39]: I would probably take this further by doing the following: 
 - Omitting `children` from the props, e.g. `Omit<RACButtonProps, "children">
 - Wrapping an icon inside the component — I think `faQuestionCircle` is the right icon, if memory serves

File: src/tooltip/styles.css.ts [hunk 3]
```diff
@@ -285,3 +285,75 @@ export const tooltipCSS = recipe<ReactAriaRecipe<TooltipRenderProps>>({
...
+        width: "100%",
+    }),
+    withPrefersMotion({
+        transitionDuration: vars.transitionDuration.short,
+        transitionProperty: "background, color",
+        transitionTimingFunction: vars.ease.quart_in_out,
+    }),
+    {
+        selectors: {
+            /**
```
Comment (alex-mcgovern) [inline src/tooltip/styles.css.ts:311]: This first-of-type/last-of-type styling is not needed

File: src/tooltip/styles.css.ts [hunk 4]
```diff
@@ -285,3 +285,75 @@ export const tooltipCSS = recipe<ReactAriaRecipe<TooltipRenderProps>>({
...
+             * Whether the button is disabled.
+             */
+            "&[data-disabled]": {
+                opacity: 0.5,
+            },
+            /**
+             * Whether the button is keyboard focused.
+             */
+            "&[data-focus-visible]": {
+                background: vars.color.bg_button_secondary_active,
```
Comment (alex-mcgovern) [inline src/tooltip/styles.css.ts:331]: I suspect that we may not want a background color here — but let's leave it in for now and assess once deployed

File: src/tooltip/index.tsx (hunk 1 above)
Comment (lordbyronknox) [reply src/tooltip/index.tsx:34]: Sweet - that's what I had it as originally.

**Pull request merged**